pkg/apis/picchu/v1alpha1/common.go

```diff
@@ -61,6 +61,7 @@ type PortInfo struct {
 	ContainerPort int32           `json:"containerPort,omitempty"`
 	Protocol      corev1.Protocol `json:"protocol,omitempty"`
 	Mode          PortMode        `json:"mode"`
+	HttpsRedirect bool            `json:"httpsRedirect,omitempty"`
```

Comment:
is there a case in which this would be false?

Reply:
1. Since picchu is meant to be an open source project, maybe. 
2. I want to make sure this doesn't break anything before I enable everywhere.